SingularityS3Uploader/src/main/java/com/hubspot/singularity/s3uploader/SingularityS3Uploader.java

```diff
@@ -182,4 +188,21 @@ private void uploadSingle(int sequence, Path file) throws Exception {
     LOG.info("{} Uploaded {} in {}", logIdentifier, key, JavaUtils.duration(start));
   }
 
+  public static boolean fileOpen(Path path) {
+    try {
+      SimpleProcessManager lsof = new SimpleProcessManager(LOG);
+      List<String> cmd = ImmutableList.of("lsof", "|", "grep", path.toAbsolutePath().toString());
```

Comment:
i think we can get away with just `("lsof", path.toAbsolutePath().toString())` (http://www.thegeekstuff.com/2012/08/lsof-command-examples/, 2nd item)